vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1043,50 +1042,56 @@ public void updateTabStyle(final int primaryColor,
                                final int fabColor,
                                final int fabPressedColor) {
         // Apply primary color
-        mToolbar.setBackgroundColor(primaryColor);
+        mToolbar.setBackgroundResource(R.drawable.act_gradient);
+        //mToolbar.setBackgroundColor(primaryColor);
```

Comment:
Please explain why this is commented out.

Reply:
I left that so that we can have a reference that was in the original app and applying primaryColor is the best practice for the toolbar.

Reply:
Can you please add this explanation as a code comment

Reply:
done